CRM/Utils/System/Standalone.php

```diff
@@ -213,9 +213,7 @@ public function url(
    */
   public static function currentPath() {
     $path = parse_url($_SERVER['REQUEST_URI'], PHP_URL_PATH);
-    if ($path[0] == '/') {
-      $path = substr($path, 1);
-    }
+    $path = trim($path, '/');
     return $path;
```

Comment:
PHPStorm would nitpick this as a useless assignment. Could just be
```suggestion
    return trim($path, '/');
```

Reply:
Fair enough